<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWebDev</class>
 <widget class="QMainWindow" name="QWebDev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWebDev</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewTextFile"/>
     <addaction name="actionNewHtmlFile"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionW3Schools"/>
    <addaction name="actionDaniweb"/>
    <addaction name="actionW3Consortium"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionValidateWebpage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewTextFile">
   <property name="text">
    <string>Text File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close Current File</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview Webpage</string>
   </property>
  </action>
  <action name="actionValidateWebpage">
   <property name="text">
    <string>Validate Webpage</string>
   </property>
  </action>
  <action name="actionW3Schools">
   <property name="text">
    <string>W3Schools</string>
   </property>
  </action>
  <action name="actionDaniweb">
   <property name="text">
    <string>Daniweb</string>
   </property>
  </action>
  <action name="actionW3Consortium">
   <property name="text">
    <string>W3 Consortium</string>
   </property>
  </action>
  <action name="actionNewHtmlFile">
   <property name="text">
    <string>HTML File</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
